backend/src/LibExecution/Interpreter.fs

```diff
@@ -86,22 +86,75 @@ let rec eval' (state : ExecutionState) (st : Symtable) (e : Expr) : DvalTask =
     | ECharacter (_id, s) -> return DChar s
 
     | ELet (id, pattern, rhs, body) ->
-      match pattern with
-      | LPVariable (_id, lhs) ->
-        if lhs = "" then
-          let! _ = preview st rhs
-          return DError(sourceID id, "Variable name in `let` is empty")
+      /// Returns `incomplete` traces for subpatterns of an unmatched pattern
```

Comment:
Hard to tell if this works from a read of it, but there's a test case for matchPatterns that could be copied to ensure the analysis works.